Scenario/scenario_shacl.ttl

```diff
@@ -0,0 +1,439 @@
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node scenario:GeneralShape ;
+            sh:order 0 ;
+            sh:path scenario:General ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node scenario:Data_SourcesShape ;
+            sh:order 6 ;
+            sh:path scenario:Data_Sources ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node scenario:StructuralShape ;
+            sh:order 4 ;
+            sh:path scenario:Structural ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node scenario:TrafficShape ;
+            sh:order 3 ;
+            sh:path scenario:Traffic ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node scenario:ParticipantsShape ;
+            sh:order 1 ;
+            sh:path scenario:Participants ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node scenario:FormatShape ;
+            sh:order 5 ;
+            sh:path scenario:Format ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node scenario:GeoreferenceShape ;
+            sh:order 7 ;
+            sh:path scenario:Georeference ] ;
+    sh:targetClass scenario:scenario .
+
+scenario:GeoreferenceShape a sh:NodeShape ;
+    sh:property [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node scenario:ProjectLocationShape ;
+            sh:order 0 ;
+            sh:path scenario:ProjectLocation ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node scenario:GeodeticReferenceSystemShape ;
+            sh:order 1 ;
+            sh:path scenario:GeodeticReferenceSystem ] ;
+    sh:targetClass scenario:georeference .
+
+scenario:GeodeticReferenceSystemShape a sh:NodeShape ;
+    sh:property [ skos:example "Lat: 1.234; Lon 45.23" ;
+            sh:description "World coordinates of map origin"@en ;
+            sh:message "Validation of origin failed!"@en ;
+            sh:name "origin"@en ;
+            sh:node scenario:Coordinate2DShape ;
+            sh:minCount 1 ;
+            sh:maxCount 1 ;
+            sh:order 2 ;
+            sh:path scenario:origin ],
+        [ skos:example "EPSG 32632" ;
+            sh:datatype xsd:string ;
+            sh:description "EPSG code of the map"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of coordinateSystem failed!"@en ;
+            sh:name "coordinateSystem"@en ;
+            sh:order 0 ;
+            sh:path scenario:coordinateSystem ],
+        [ skos:example "EPSG xyz" ;
+            sh:datatype xsd:string ;
+            sh:description "Ellipsodial height or orthometric height"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of heightSystem failed!"@en ;
+            sh:name "heightSystem"@en ;
+            sh:order 1 ;
+            sh:path scenario:heightSystem ] ;
+    sh:targetClass scenario:GeodeticReferenceSystem .
+
+scenario:BoundingBoxShape a sh:NodeShape ;
```

Comment:
I have seen the BoundingBoxShape and other Shapes in this file in several other domains like hdmap, too. 
Are they all the same? If yes we should centralize them.
Disadvantage: creating claims the the SD-Creation-Wizard will not be that easy anymore because the Wizard will not know it. The Catalogue does because it loads all Shapes when validating

Reply:
I have created an issue for that, s. https://github.com/GAIA-X4PLC-AAD/ontology-management-base/issues/34
@3Dbastian will take a look into this topic. I would also prefer to extract those contents into a separate shape which can be used by multiple data categories. Might be also of advantage for the data search later, @lenasauermann.